File: src/assets/usage-examples-code/windows/current_window/current_window-1-example.cpp
#include "splashkit.h"

int main()
{
    // Open two windows
    window window1 = open_window("Window 1", 800, 600);
    window window2 = open_window("Window 2", 800, 600);

    // Display a message and wait for a few seconds
    write_line("Activating and then retrieving the current window in 5 seconds...");
    delay(5000);

    // Retrieve the current active window by switching context
    window current = current_window();

    // Print the title of the current active window
    write_line("The current active window is: " + window_caption(current));

    // Close the windows
    close_window(window1);
    close_window(window2);

    return 0;
}

File: src/assets/usage-examples-code/windows/current_window/current_window-1-example.cs
using static SplashKitSDK.SplashKit;  
using SplashKitSDK;  

// Open two windows
Window window1 = OpenWindow("Window 1", 800, 600);
Window window2 = OpenWindow("Window 2", 800, 600);

// Display a message and wait for a few seconds
WriteLine("Activating Window 2 and then retrieving the current window in 5 seconds...");
Delay(5000);

// Retrieve the current active window by switching context
Window current = CurrentWindow();

// Print the title of the current active window
WriteLine("The current active window is: " + WindowCaption(current));

// Close the windows
CloseWindow(window1);
CloseWindow(window2);

